Jenkinsfile

```diff
@@ -56,5 +62,29 @@ pipeline {
         }
       }
     }
+
+    stage('docker') {
+      steps {
+        container('img') {
+          script {
+          def tag = null
+          if (env.BRANCH_NAME == 'master') {
+            tag = 'latest'
+          } else {
+            tag = env.GIT_COMMIT
+          }
+            sh "img build -t ${image}:${tag} ."
```

Comment:
Will we actually push it to ECR for our own use?

Reply:
how you would test  image after build in case of changes?